web/src/main/java/com/synopsys/integration/alert/web/api/job/JobConfigActions.java

```diff
@@ -280,7 +294,25 @@ protected ValidationActionResponse validateWithoutChecks(JobFieldModel resource)
         List<AlertFieldStatus> fieldStatuses = new ArrayList<>();
 
         validateJobNameUnique(jobId, resource).ifPresent(fieldStatuses::add);
-        fieldStatuses.addAll(fieldModelProcessor.validateJobFieldModel(resource));
+
+        boolean validateWithDescriptorValidators = shouldValidateWithDescriptorValidators(resource);
+
+        if (validateWithDescriptorValidators) {
+            fieldStatuses = resource.getFieldModels()
+                .stream()
+                .map(FieldModel::getDescriptorName)
+                .map(descriptorMap::getDescriptorKey)
+                .flatMap(Optional::stream)
+                .map(descriptorMap::getDescriptor)
+                .flatMap(Optional::stream)
+                .map(Descriptor::getDistributionValidator)
+                .flatMap(Optional::stream)
+                .map(validator -> validator.validate(resource))
+                .flatMap(Collection::stream)
+                .collect(Collectors.toList());
```

Comment:
duplicated on line 356. We should add a private method for this.

Reply:
Extracted!